syntax/javascript.vim

```diff
@@ -324,8 +324,8 @@ if version >= 508 || !exists("did_javascript_syn_inits")
   HiLink jsModuleKeywords       Include
   HiLink jsModuleOperators      Include
   HiLink jsModuleDefault        Include
-  HiLink jsDecorator            Special
-  HiLink jsDecoratorFunction    Special
+  HiLink jsDecorator            Tag
+  HiLink jsDecoratorFunction    Tag
```

Comment:
I think `jsDecoratorFunction` should probable be linked to `Function` or `Identifier`, then `jsDecorator` should perhaps be linked to `Operator`, `Label`, `Type`, or `Special`